src/getHumanAge.test.js

```diff
@@ -7,4 +7,52 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should have right length', () => {
+    const result = getHumanAge(2, 3);
+
+    expect(result).toHaveLength(2);
+  });
+
+  test('should return an Array', () => {
+    const result = getHumanAge;
+
+    expect(Array.isArray(result));
+  });
+
+  test('should have expected values', () => {
+    const result = getHumanAge(0, 0);
+
+    expect(result).toEqual([0, 0]);
+  });
+
+  test('should have expected values bv1', () => {
+    const result = getHumanAge(5, 5);
+
+    expect(result).toEqual([0, 0]);
+  });
+
+  test('should have expected values bv1', () => {
+    const result = getHumanAge(15, 15);
+
+    expect(result).toEqual([1, 1]);
+  });
+
+  test('should have expected values bv1', () => {
+    const result = getHumanAge(16, 16);
+
+    expect(result).toEqual([1, 1]);
+  });
```

Comment:
The test cases for expected values seem to follow a pattern, but the descriptions are not unique. Ensure each test case has a unique description to clearly indicate what specific scenario or boundary value it is testing.